### PR TITLE
Added text for appendix c

### PR DESCRIPTION
**Pull request by talltree**
This text was from the Google doc that has been reviewed separately: https://docs.google.com/document/d/1Lkd7G4fw7ruJqB_Knziu7_ziq2nRINcXE4p8P9WM60k/edit?usp=sharing

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -584,5 +584,25 @@ For ease of reference, the following table consolidates all normative requiremen
...
+* Multiple QR formats and out-of-band introduction (OOBI) protocols (e.g., [OASIS Secure QR Codes](https://www.oasis-open.org/2022/07/12/secure-qr-code-authentication-v1-0-from-esat-tc-approved-as-a-committee-specification/)).
+
+From a ToIP perspective, all of these are potential component specifications.
+
+* Some may fully meet the requirements of the ToIP Technology Architecture Specification without any modification. 
+* Others may require ToIP-conformant profiles to be written.
+* Still others may require (hopefully small) revisions to meet ToIP requirements.
+* A few (ideally very few) may need to be developed from scratch, either at the ToIP Foundation or another suitable industry organization.
+
+Given the speed at which this new evolutionary branch of the Internet is evolving, the ToIP Foundation is maintaining a “mapping” of existing technologies and open standards into the ToIP stack in a separate living document called [Evolution of the ToIP Stack](https://trustoverip.org/our-work/evolution-of-the-toip-stack/).
```
Comment (a-fox) [inline spec.md:606]: Have we confirmed / are we sure about the URL? Either I can't confirm it yet (not yet available), or it's wrong. Please confirm @talltree.

Comment (talltree) [reply spec.md:606]: Antti, you are indeed checking closely! The reason that URL does not resolve yet is that the page will not be created until tomorrow when the announcement goes live. You can see the full task list together with all the pending links, pages and page copy in this Google doc: https://docs.google.com/document/d/1g7cmPMzmq5G9OKHyu54tfjpMoCD7ihXe2jLRBXJFGLk/edit?usp=sharing

So this should not be a blocker to merging.

Comment (a-fox) [reply spec.md:648]: Great, thanks this is good to go then1

**Comment by a-fox** [comment]
Reviewed and approved!

**Pull request merged**